app/hotels/README.md

```diff
@@ -1,7 +1,7 @@
 ```js
 import { HotelsStandalonePackage } from '@kiwicom/react-native-app-hotels';
 
-const render = () => <HotelsStandalonePackage
+const render() <HotelsStandalonePackage
```

Comment:
This does not work here, even though it's just a README file

Reply:
Hehe, I was pretty sure I undid that change 🙈